include/language-factory.php

```diff
@@ -98,6 +98,30 @@ public static function get_from_terms( array $terms ) {
 		$flag_props = self::get_flag( $data['flag_code'], $data['name'], $data['slug'], $data['locale'] );
 		$data       = array_merge( $data, $flag_props );
 
+		$default = array(
+			'home_url'   => '',
+			'search_url' => '',
+		);
+
+		/**
+		 * Filters the home URL and the search URL before the language is created.
+		 *
+		 * @since 3.4
+		 *
+		 * @param array  $default {
+		 *     Array of filterable language data.
+		 *
+		 *     @type string $home_url   Language home URL.
+		 *     @type string $search_url Language search URL.
+		 * }
+		 * @param array $data Language data.
+		 *
+		 * @phpstan-param array{home_url: string, search_url: string} $default
+		 * @phpstan-param non-empty-array<string, mixed> $data
+		 */
+		$_default = apply_filters( 'pll_filterable_language_data', $default, $data );
```

Comment:
You can keep the same variable for the filtered parameter and returned value.
Suggesting `pll_language_data` or `pll_additional_language_data` as filter name.

Reply:
Not sure I can use the same variable name with `array_intersect_key()` 🤔.
Filter renamed in 42de22a.